### PR TITLE
enums.py  __init__.py  programme.py  student.py

### PR DESCRIPTION
**Pull request by Thenebit**
So first of all,  i updated the student.py by including gender and creating an enum.py to contain a predefined data which will be used.  Also, i took to myself of helping @360konet with the programme model when he was getting errors. Even though he has now been able to create it, i think to be safe some data columns were just made as str.

**Issue**

- Link:
- Summary:

**How**

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: app/models/student.py [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
-    nationality: str
+    nationality: Nationality
+    gender: Gender
     age: int
     place_of_birth: str
     current_residence: str
     email: str = Field(unique=True, index=True)
     index_number: str
+
+    programme: List[Programme] = Relationship(back_populates="students")
```
Comment (Fourteen98) [inline app/models/student.py:24]: you just gave it a one to many relationship

File: app/models/enums.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+
+import pycountry
+
+
+class Nationality(str, Enum):
+    @classmethod
+    def choices(cls):
+        return [(country.alpha_2, country.name)
```
Comment (Fourteen98) [inline app/models/enums.py:9]: what happens if not found, an exception is thrown but u did not handle that

**Comment by Fourteen98** [comment]
@Thenebit , this has been fixed and merged,

File: app/models/student.py (hunk 1 above)
Comment (Thenebit) [reply app/models/student.py:24]: understood

File: app/models/enums.py (hunk 2 above)
Comment (Thenebit) [reply app/models/enums.py:9]: did not even think about that. But it makes since to have it

**Comment by Thenebit** [comment]
Thanks for the feedback

**Comment by Fourteen98** [comment]
> Thanks for the feedback

you welcome

**Pull request closed without merging**